### ISSUE TITLE
Feature request: add a per-wallet config option to disable BIP 69 sorting

### ISSUE THREAD

**Issue by A60AB5450353F40E** [Electron-Cash/Electron-Cash#2623]
Useful for some of us who may need ins and outs to have specific slots, like when adding covenants to unsigned TX, or when spending an authchain together with a BCMR OP_RETURN - in which case we want the authchain p2pkh to go to slot0 to preserve it and OP_RETURN in some other slot.

**Comment by imaginaryusername** [comment]
I don't know the inner structure of how it works currently, but would it not be better to allow some sort of accessible configs to allow per-tx exceptions instead? If this is per-wallet it'll limit conflicting-with-bip69 usecases like BCMR to segregated wallets only, which seems quite constrictive not just for this case but also for future potential.

**Comment by cculianu** [comment]
It would be possible but I think less work for me just to make global per-wallet option tbh. Like writing code to examine the txn and testing it.. is a lot of hours.

Also can offer checkbox on the send tab as well as the token-send tab to toggle it on/off at user's behest on a per-tx basis.... that might be a happy compromise.

**Comment by cculianu** [comment]
Yes it would clutter it up. Understood about not wanting to clutter it.

An alternative is to make the txdialog (“Preview”) window offer it as a checkbox in some circumstances..

**Comment by jonas-lundqvist** [comment]
How should the change output be handled? Either it has to be added as the last output or the user has to explicitly specify it somehow.

Is there a need for a user to somehow control the order of the inputs also?

**Comment by jonas-lundqvist** [comment]
> If this is per-wallet it'll limit conflicting-with-bip69 usecases like BCMR to segregated wallets only, which seems quite constrictive not just for this case but also for future potential.

I assume he meant the (toggleable) per-wallet preferences, in the same place as "Sign with Schnorr signatures" – not that it's a permanent setting for the wallet. That is IMHO where this setting belongs so a niche power-user that wants to do this could toggle it on the (supposedly?) rare occasion it is needed.

**Comment by cculianu** [comment]
My idea was that if it's a tx originating in the send tab.. the `TxDialog` (`TxDialog` = that window you get if you hit `Preview`) could offer a checkbox to "unsort" it... (`TxDialog` would receive 2 copies of the tx secretly from the send tab, 1 that is the sorted one and one that is unsorted).  If the `TxDialog` is created with such a pair of txns, user will see the checkbox. Then user can toggle/play with the checkbox and see the effects it has on the txn immediately.
